.storybook/manager.js

```diff
@@ -0,0 +1,6 @@
+import { addons } from '@storybook/manager-api'
+import { themes } from '@storybook/theming'
+
+addons.setConfig({
+  theme: themes.light
```

Comment:
without this, the new default for Storybook is 'auto', which looks weird next to our non-dark-mode-friendly design system. I also didn't want to change too much while Iana is away 😅

<img width="1378" alt="Screenshot 2023-08-09 at 14 23 36" src="https://github.com/archilogic-com/honeycomb/assets/1694288/034586ac-d952-42ca-879a-875b4a0e0c18">